packages/turbo-codemod/src/transforms/rename-output-mode.ts

```diff
@@ -0,0 +1,91 @@
+import path from "node:path";
+import { readJsonSync, existsSync } from "fs-extra";
+import { type PackageJson, getTurboConfigs } from "@turbo/utils";
+import type { Schema as TurboJsonSchema, OutputMode } from "@turbo/types";
+import type { TransformerArgs } from "../types";
+import { getTransformerHelpers } from "../utils/getTransformerHelpers";
+import type { TransformerResults } from "../runner";
+
+// transformer details
+const TRANSFORMER = "rename-output-mode";
+const DESCRIPTION =
+  'Rename the "outputMode" key to "outputLogs" in `turbo.json`';
+const INTRODUCED_IN = "2.0.0";
+
+function migrateConfig(config: TurboJsonSchema) {
+  for (const [_, taskDef] of Object.entries(config.pipeline)) {
+    if (Object.prototype.hasOwnProperty.call(taskDef, "outputMode")) {
+      //@ts-expect-error - outputMode is no longer in the schema
+      taskDef.outputLogs = taskDef.outputMode as OutputMode;
+      //@ts-expect-error - outputMode is no longer in the schema
+      delete taskDef.outputMode;
+    }
+  }
+
+  return config;
+}
+
+export function transformer({
+  root,
+  options,
+}: TransformerArgs): TransformerResults {
+  const { log, runner } = getTransformerHelpers({
+    transformer: TRANSFORMER,
+    rootPath: root,
+    options,
+  });
+
+  // If `turbo` key is detected in package.json, require user to run the other codemod first.
+  const packageJsonPath = path.join(root, "package.json");
+  // package.json should always exist, but if it doesn't, it would be a silly place to blow up this codemod
+  let packageJSON = {};
+
+  try {
+    packageJSON = readJsonSync(packageJsonPath) as PackageJson;
+  } catch (e) {
+    // readJSONSync probably failed because the file doesn't exist
```

Comment:
I think we would want to abort if we are missing/can't read the root `package.json`

Reply:
I copied this logic from the outputs codemod. I'm down to change it though